recipes/structlog/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "16.1.0" %}
+{% set sha256 = "46883e8a48918f18ab99c6413e166dc544acd1ad8bba659d152ac6e82c81c2ca" %}
+
+package:
+  name: structlog
+  version: {{ version }}
+
+source:
+  fn: structlog-{{ version }}.tar.gz
+  url: https://github.com/hynek/structlog/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - structlog
+
+about:
+  home: https://github.com/alecthomas/structlog
```

Comment:
According to PyPI it is this `http://www.structlog.org/`. Let's go with this last one.